### ISSUE TITLE
Apply remote connection button color to all theme colors

### ISSUE THREAD

**Issue by ambidexterich** [ambidexterich/vscode-fresh-material#16]

**Comment by tyler36** [comment]
Not sure if I'm doing something wrong ... 

- Clone repo
- Switch to `develop` banch
- Generate package
```bash
vsce package
# produces fresh-material-2.3.10
```
- Install from VSIX
- Restart VSCode

## Results:
### Blue theme:
- white remote backgrade
- no hover
![Fresh-material-blue](https://user-images.githubusercontent.com/7234392/118572374-420f5300-b7bb-11eb-9f4f-d085341c9d2b.png)

### Cyan
- `dirty` cyan color
![Fresh-material-cyan](https://user-images.githubusercontent.com/7234392/118572389-4e93ab80-b7bb-11eb-83e9-33735cb7b8cc.png)
- *very* subtle hover
![Fresh-material-cya-hover](https://user-images.githubusercontent.com/7234392/118572406-54898c80-b7bb-11eb-83ac-c30f7101d723.png)

### Green
- `dirty` green color
![Fresh-material-green](https://user-images.githubusercontent.com/7234392/118572425-5eab8b00-b7bb-11eb-91d1-13462fe5ec4f.png)
- *very* subtle hover
![Fresh-material-green-hover](https://user-images.githubusercontent.com/7234392/118572439-6703c600-b7bb-11eb-873d-6d0107e62bbd.png)

### Indigo theme:
- white remote backgrade
- no hover
![Fresh-material-indigo](https://user-images.githubusercontent.com/7234392/118572362-3a4fae80-b7bb-11eb-9c97-e92b4ef5ddbd.png)

### Pink theme
- cyan background with green text
![Fresh-material-pink](https://user-images.githubusercontent.com/7234392/118572464-7551e200-b7bb-11eb-8271-4de2136ac14a.png)

### Red theme
- blue background with green text
![Fresh-material-red](https://user-images.githubusercontent.com/7234392/118572476-7b47c300-b7bb-11eb-9e27-e33a324d50c4.png)

**Comment by tyler36** [comment]
I used a [portabe copy of VS Code](https://code.visualstudio.com/docs/editor/portable) with the [Remote SSH](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh) extension for testing.

I don't think you need to do anything other setup on your machine to install the extension so you should be able to do testing.
I'm happy to help too.

**Comment by ambidexterich** [comment]
Thanks, those are correct for now. I tweaked the foreground color to make it standout more but VSCode uses the same colors as the badges which is the secondary color. They don't provide a way to change the hover or active color so it uses the same hover color as everything in the status bar. I used a transparency which is why you see a hover on some things and not others. Unfortunately that will be the way they are until MS provides more theming options.

I'm working on revamping the entire theme to try to deal with things that vscode doesn't let you style specifically.
